htdocs/themes/math4/math4.css

```diff
@@ -122,16 +125,12 @@ legend {
 }
 
 #loginstatus {
-        padding-top:.5ex;
-        padding-right: .5em;
+	padding: 5px 5px 5px 0;
 	color: white;
 	text-align: right;
 	font-size: 0.85em;
 	font-weight: normal;
 } 
-#loginstatus a.btn {
-    margin-bottom:.5ex;
```

Comment:
Sometimes there are two buttons. One for logging yourself out, and one to stop acting as another user. With no margin-bottom on the button, the two buttons are right up against each other which looks bad.

Everything else seems good, so I will merge and then do something to separate the buttons.

Reply:
I forgot about that case.  Yeah, that doesn't look the best.